agile-development/scrum.md

```diff
@@ -19,6 +19,12 @@ There is so much to learn, and practice everyday. Decision fatigue is a matter o
 
 Practicing scrum will enable you and your team to instantly address change requests, and establish better connection between business and development teams. While this may not be a relevant experience at the moment, once you begin working you may observe that information silos between business and development teams are real. And, using Scrum promotes team morale, to offset the lack of communication, while introducing predictability and transparency when coupled with sprints. Not only that, using scrum will allow you to better predict the success of a project early on.
 
+**Which companies use Scrum?**
+- The links mentioned below describes the use of Scrum in these companies:
+  - [3M](https://www.quickstart.com/blog/how-agile-scrum-training-transformed-these-5-companies/): Minnesota Mining and Manufacturing Company.
```

Comment:
Since it's the same link for all 3, can you change this description to being about finding the 3 companies at this site? and then link it once.